TealiumFacebookExample/TealiumFacebookExample/SceneDelegate.swift

```diff
@@ -38,6 +39,13 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     }
 
     func sceneDidBecomeActive(_ scene: UIScene) {
+        if #available(iOS 14, *) {
+            ATTrackingManager.requestTrackingAuthorization(completionHandler: { status in
```

Comment:
Using Xcode 12.5.1 the example crashes at launch. 
**NSUserTrackingUsageDescription** is missing in the info plist of the example.